<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QG_TrimAmountOptions</class>
 <widget class="QWidget" name="QG_TrimAmountOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>110</width>
    <height>24</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>110</width>
    <height>22</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>150</width>
    <height>24</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Trim Amount Options</string>
  </property>
  <property name="toolTip">
   <string>Distance. Negative values for trimming, positive values for extending.</string>
  </property>
  <layout class="QHBoxLayout">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="margin">
    <number>1</number>
   </property>
   <item>
    <widget class="QLabel" name="lDist">
     <property name="minimumSize">
      <size>
       <width>51</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Length:</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="leDist">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>40</horstretch>
       <verstretch>22</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>22</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>22</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Distance. Negative values for trimming, positive values for extending. Negative sign is ignored when trimming to final total length.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbTotalLength">
     <property name="minimumSize">
      <size>
       <width>8</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="toolTip">
      <string>The input length is used as total length after trimming, instead of length increase</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="sep1">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>2</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>2</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::VLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>leDist</sender>
   <signal>textChanged(QString)</signal>
   <receiver>QG_TrimAmountOptions</receiver>
   <slot>updateDist(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
